rdl/data_sources/MsSqlDataSource.py

```diff
@@ -193,6 +193,7 @@ def init_change_tracking(self, table_config, last_known_sync_version):
             f" CASE WHEN @last_known_sync_version >= @min_valid_version THEN 1 ELSE 0 END; \n" \
             f"DECLARE @last_sync_version                BIGINT; \n" \
             f"DECLARE @force_full_load                  BIT; \n" \
+            f"DECLARE @data_changed_since_last_sync     BIT; \n" \
```

Comment:
can we update the commentary on line 185 to include the new value (`data_changed_since_last_sync`) being computed?